src/Compilers/Core/Portable/PEWriter/MetadataVisitor.cs

```diff
@@ -412,6 +413,7 @@ public virtual void Visit(IParameterDefinition parameterDefinition)
             Debug.Assert((marshalling != null || !parameterDefinition.MarshallingDescriptor.IsDefaultOrEmpty) == parameterDefinition.IsMarshalledExplicitly);
 
             this.Visit(parameterDefinition.GetAttributes(Context));
+            this.Visit(parameterDefinition.RefCustomModifiers);
```

Comment:
>this.Visit(parameterDefinition.RefCustomModifiers); [](start = 12, length = 51)

It looks like there is one more place in ReferenceDependencyWalker.cs that should be adjusted in a similar way. And in ```public virtual void Visit(IParameterTypeInformation parameterTypeInformation)``` in this type as well.